OpenSim/Sandbox/MatlabScripts/createActuatorsFile.m

```diff
@@ -0,0 +1,156 @@
+function createActuatorsFile
```

Comment:
Perhaps accept optional argument containing path/filename of model (e.g., so this function could be used in a loop without prompting the user with a dialog box).